hawc/apps/assessment/views.py

```diff
@@ -983,36 +987,57 @@ def read(self, request: HttpRequest, *args, **kwargs):
     @action(methods=("get", "post"), permission=can_edit)
     def create(self, request: HttpRequest, *args, **kwargs):
         template = self.form_fragment
+        kwargs = {}
+        retarget_form = False
         if request.method == "POST":
             form = forms.LabelForm(request.POST, assessment=request.item.assessment)
             if form.is_valid():
                 self.perform_create(request.item, form)
-                template = self.detail_fragment
+                template = self.list_fragment
+                kwargs = {"object_list": self.get_queryset(request)}
+            else:
+                retarget_form = True
         else:
             form = forms.LabelForm(assessment=request.item.assessment)
-        context = self.get_context_data(form=form)
-        return render(request, template, context)
+        context = self.get_context_data(form=form, **kwargs)
+        response = render(request, template, context)
+        if retarget_form:
+            response["HX-Retarget"] = "#label-edit-row-"
```

Comment:
didn't know about this, thanks, so basically on success it uses the target in the template, but if there's a form validation error it retargets and replaces itself. 

https://htmx.org/reference/

> a CSS selector that updates the target of the content update to a different element on the page